exercises/ex1.cpp

```diff
@@ -11,38 +9,39 @@ int main()
   cout << "Enter week number(1-7): " << endl;
   cin >> week;
 
-  if (week == 1)
-  {
-    cout << "Monday" << endl;
-  }
-  else if (week == 2)
-  {
-    cout << "Tuesday" << endl;
-  }
-  else if (week == 3)
-  {
-    cout << "Wednesday" << endl;
-  }
-  else if (week == 4)
-  {
-    cout << "Thursday" << endl;
-  }
-  else if (week == 5)
-  {
-    cout << "Friday" << endl;
-  }
-  else if (week == 6)
-  {
+switch (week) {
+case 1:
+   cout << "Monday" << endl;
+   break;
+
+case 2:
+   cout << "Tuesday" << endl;
+   break;
+
+case 3:
+   cout << "Wednesday" << endl;
+   break;
+
+case 4:
+   cout << "Thursday" << endl;
+   break;
+
+case 5:
+   cout << "Friday" << endl;
+   break;
+
+case 6:
     cout << "Saturday" << endl;
-  }
-  else if (week == 7)
-  {
+   break;
+
+case 7:
     cout << "Sunday" << endl;
-  }
-  else
-  {
-    cout << "Invalid input! Please enter week number between 1-7." << endl;
-  }
+   break;
+
+default:
+   cout << "Invalid input! Please enter week number between 1-7." << endl;
+   break;
+}
 
   return 0;
```

Comment:
Il `return 0` è poco esplicativo di per sè.
Potresti fare un:
`#define EXIT_SUCCESS 0` e inserire alla fine del main `return EXIT_SUCCESS;` in modo tale da incrementare la leggibilità del codice.

```suggestion
  return EXIT_SUCCESS;
```